tf-modules/s3-replicator/variables.tf

```diff
@@ -1,11 +1,16 @@
 
+variable "security_group_ids" {
+  type    = list(string)
+  default = []
```

Comment:
I typically set the default to `null` and then use `var.security_group_ids == null ? blah : other_blah`